cfc-testing/README.md

```
@@ -59,29 +59,124 @@ java -cp "target/classes:target/dependency/*" com.cloudforgeci.samples.app.Inter
 
 ## Sample Applications
 
-For production sample applications that demonstrate how to use CloudForge Community, see the **`cloudforge-sample`** project at `/Users/phillip/projects/cloudforge-sample`.
+For production sample applications that demonstrate how to use CloudForge Community, see the **`cloudforge-sample`** repository at https://github.com/CloudForgeCI/cloudforge-sample.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)